src/tts/magpietts/model.cpp

```
@@ -721,6 +721,8 @@ magpietts_model_load_impl(
     h.mask_token_id = gguf_i32(model.gguf, "magpietts.mask_token_id", h.mask_token_id);
     h.frame_stacking_factor =
         gguf_i32(model.gguf, "magpietts.frame_stacking_factor", h.frame_stacking_factor);
+    const int32_t stored_stacked_codebooks =
```

Clang (14.0.6)

[warning] variable 'stored_stacked_codebooks' is not initialized

(cppcoreguidelines-init-variables)